src/components/Tag/Tag.module.css

```diff
@@ -48,7 +48,8 @@
  */
 .tag--neutral {
   --tag-primary-color: var(--eds-theme-color-text-neutral-default);
-  --tag-secondary-color: var(--eds-theme-color-background-neutral-medium);
+  /* TODO: use background-neutral-medium when new colors land */
+  --tag-secondary-color: var(--eds-theme-color-background-neutral-subtle);
```

Comment:
new grays seem to be generally a little lighter than the current grays; `neutral-medium` looked too dark with the existing colors